Road/notebooks/data_cleaning.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # from source.constants import PUNCT_TO_REMOVE
</pre><p>No need for this line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Road-Status/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>